runtime/common/src/lib.rs

```diff
@@ -70,10 +61,7 @@ pub const EXISTENTIAL_DEPOSIT: Balance = MILLIUNIT;
 // Async backing
 /// Maximum number of blocks simultaneously accepted by the Runtime, not yet included
 /// into the relay chain.
-#[cfg(not(feature = "paseo"))]
 pub const UNINCLUDED_SEGMENT_CAPACITY: u32 = 3;
```

Comment:
Feels a little weird introducing async backing to Paseo in a PR for removing Rococo.

Reply:
reverted